src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="searchbar"
+      data-qa="big"
+    >
```

Comment:
HTML5 does not support self-closing form tags. You should use separate opening and closing tags for the form element, like `<form>...</form>`.